README.md

```diff
@@ -236,8 +236,9 @@ _bountydetails function returns the following fields:
 git clone https://github.com/gitcoinco/web.git
 cd web
 cp app/app/local_settings.py.dist app/app/local_settings.py
-docker-compose up
+docker-compose up -d
 ```
+Navigate to `http://0.0.0.0:8000/`.
```

Comment:
LGTM... althought not necessarily related to this PR

Reply:
If its ok with you, I'll leave it in. I think this PR will be ready to merge 🌐 in soon.

Reply:
ok w me